ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -62,17 +61,26 @@ public AmbrySecurityService(FrontendConfig frontendConfig, FrontendMetrics front
     } else if (restRequest == null) {
       throw new IllegalArgumentException("RestRequest is null");
     }
-    FutureResult<Void> futureResult = new FutureResult<Void>();
-    if (callback != null) {
-      callback.onCompletion(null, exception);
-    }
-    futureResult.done(null, exception);
     frontendMetrics.securityServiceProcessRequestTimeInMs.update(System.currentTimeMillis() - startTimeMs);
-    return futureResult;
+    callback.onCompletion(null, exception);
   }
 
   @Override
-  public Future<Void> processResponse(RestRequest restRequest, RestResponseChannel responseChannel, BlobInfo blobInfo,
+  public void postProcessRequest(RestRequest restRequest, Callback<Void> callback) {
+    Exception exception = null;
+    frontendMetrics.securityServicePostProcessRequestRate.mark();
```

Comment:
May I know the reason you want to track this? Would this be `totalRequestRate - totalPutRate`?